File: Math/c/Fibonacci.c
#include<stdio.h>
#include<math.h>
int main() {
    int a = 0;
    int b = 1;
    printf("%.17f\n", 0.1 + 0.2);
    for (int i=1; i<=34; i++) {
        b += a;
        a = b - a;
        printf("%d\t%d\t%d\t:\t%.17f\n", i, a, b, (double) b / a);
        printf("\t\t\t:\t%.15f\n", round(1E15 * (double) b / a)/1E15);
    }
    return 0;
}

File: Math/Fibonacci.js
var a = 0;
var b = 1;
console.log(0.1 + 0.2);
for (var i = 1; i <= 34; i++) {
    b += a;
    a = b - a;
    console.log(i + "\t" + a + "\t\t" + b + "\t\t:\t" + (b / a));
    console.log(i + "\t\t\t\t\t\t" + (Math.round(1E15 * b / a) / 1E15));
}
